plugin/trino-postgresql/src/test/java/io/trino/plugin/postgresql/TestPostgreSqlJdbcConnectionCreation.java

```diff
@@ -0,0 +1,181 @@
+ */
+package io.trino.plugin.postgresql;
+
+import com.google.common.collect.ImmutableList;
+import com.google.inject.Binder;
+import com.google.inject.Provides;
+import com.google.inject.Singleton;
+import io.airlift.configuration.AbstractConfigurationAwareModule;
+import io.trino.plugin.jdbc.BaseJdbcConfig;
+import io.trino.plugin.jdbc.BaseJdbcConnectionCreationTest;
+import io.trino.plugin.jdbc.ConnectionFactory;
+import io.trino.plugin.jdbc.DriverConnectionFactory;
+import io.trino.plugin.jdbc.ExtraCredentialsBasedIdentityCacheMappingModule;
+import io.trino.plugin.jdbc.ForBaseJdbc;
+import io.trino.plugin.jdbc.JdbcConnectorFactory;
+import io.trino.plugin.jdbc.credential.CredentialProvider;
+import io.trino.plugin.jdbc.credential.CredentialProviderModule;
+import io.trino.plugin.jdbc.credential.StaticCredentialProvider;
+import io.trino.plugin.tpch.TpchPlugin;
+import io.trino.spi.Plugin;
+import io.trino.spi.connector.ConnectorFactory;
+import io.trino.testing.DistributedQueryRunner;
+import io.trino.testing.QueryRunner;
+import io.trino.tpch.TpchTable;
+import org.postgresql.Driver;
+import org.testng.annotations.Test;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.Properties;
+
+import static io.airlift.configuration.ConfigurationAwareModule.combine;
+import static io.airlift.testing.Closeables.closeAllSuppress;
+import static io.trino.plugin.postgresql.PostgreSqlQueryRunner.createSession;
+import static io.trino.plugin.tpch.TpchMetadata.TINY_SCHEMA_NAME;
+import static io.trino.testing.QueryAssertions.copyTpchTables;
+import static io.trino.tpch.TpchTable.NATION;
+import static io.trino.tpch.TpchTable.REGION;
+import static java.util.Objects.requireNonNull;
+import static org.postgresql.PGProperty.REWRITE_BATCHED_INSERTS;
+
+@Test(singleThreaded = true) // this class is stateful, see fields
+public class TestPostgreSqlJdbcConnectionCreation
+        extends BaseJdbcConnectionCreationTest
+{
+    @Override
+    protected QueryRunner createQueryRunner()
+            throws Exception
+    {
+        TestingPostgreSqlServer postgreSqlServer = closeAfterClass(new TestingPostgreSqlServer());
+        Properties connectionProperties = new Properties();
+        connectionProperties.put(REWRITE_BATCHED_INSERTS.getName(), "true");
+        CredentialProvider credentialProvider = new StaticCredentialProvider(
+                Optional.of(postgreSqlServer.getUser()),
+                Optional.of(postgreSqlServer.getPassword()));
+        DriverConnectionFactory delegate = new DriverConnectionFactory(new Driver(), postgreSqlServer.getJdbcUrl(), connectionProperties, credentialProvider);
+        this.connectionFactory = new ConnectionCountingConnectionFactory(delegate);
+        return createPostgreSqlQueryRunner(postgreSqlServer, ImmutableList.of(NATION, REGION), connectionFactory);
+    }
+
+    @Test
+    public void dummyTestToMakeSureTestNgRunsTestsSerially() {}
```

Comment:
See https://github.com/trinodb/trino/blob/23ca99c1ba4cbd4ca1eb786b3381c5a756126021/plugin/trino-resource-group-managers/src/test/java/io/trino/plugin/resourcegroups/db/TestDbResourceGroupsOracleFlywayMigration.java#L86-L92 for the convention.

Reply:
It is no longer needed.